config/kubernetes/staging/dashboard.yaml

```diff
@@ -0,0 +1,828 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: laa-criminal-applications-dashboard
+  namespace: laa-criminal-applications-datastore-staging
+  labels:
+    grafana_dashboard: ""
+data:
+  register-laa-criminal-applications-datastore-staging-dashboard.json: |
```

Comment:
We can have a shorter name, we don't have to add the namespace in it:
`crime-applications-datastore-dashboard.json`
Also we can have the same name in the metadata.name

The thing is, there will be only one dashboard for all namespaces, not one per-namespace/env. Will explain in another comment.